Examples/test-suite/fortran/Makefile.in

```diff
@@ -0,0 +1,68 @@
+#######################################################################
+# Makefile for Fortran test-suite
+#######################################################################
+
+LANGUAGE	= fortran
+FC      	= @FC@
+SCRIPTSUFFIX= _runme.f90
+
+FORTRANOBJEXT = o
+
+FORTRAN_SO = @FORTRAN_SO@
+FCFLAGS = @FCFLAGS@
+CCSHARED=	@CCSHARED@
+
+srcdir         = @srcdir@
+top_srcdir     = @top_srcdir@
+top_builddir   = @top_builddir@
+abs_top_srcdir = @abs_top_srcdir@
+
+CPP_TEST_CASES = \
+	fortran_bindc
+
+CPP11_TEST_CASES = \
+	fortran_shared_ptr
+
+C_TEST_CASES = \
+	fortran_array_typemap \
+	fortran_bindc_c \
+	fortran_bindc_all \
+	fortran_enums \
+
+include $(srcdir)/failing-cpp-tests.mk
+include $(srcdir)/failing-c-tests.mk
+include $(srcdir)/../common.mk
+
+.SUFFIXES: .cpptest .ctest .multicpptest
```

Comment:
Not sure why .SUFFIXES is needed for Fortran. The GNU documentation indicate this is for old fashioned and obsolete pattern rules. The 3 rules below are the modern pattern rules.

Reply:
I think this makefile was originally copied from `Examples/test-suite/go/Makefile.in`, which seems to have it as well. I removed it from the Fortran makefile.